Resources/public/css/styles.css

```diff
@@ -345,6 +345,28 @@ td.sonata-ba-list-field .editable-empty:not(.editable-open) {
     width: auto; /* See https://github.com/sonata-project/SonataAdminBundle/issues/2950 */
 }
 
+/* icheck */
+.checkbox div[class^="icheckbox"],
```

Comment:
Or maybe we are using iCheck / AdminLTE wrong ? @vincenttouzet, can you find out if it is the former or the latter? Anyway, there should indeed not be an iCheck override here.